examples/go/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/GoogleContainerTools/distroless/blob/master/examples/go
```

Comment:
Why `.../blob/master/...`? Why not `github.com/GoogleContainerTools/distroless/examples/go`?

Reply:
Seems like a copy mistake, fixed.